content/4_Gold/Intro_DP.mdx

```diff
@@ -183,7 +183,14 @@ The next few modules provide examples of some classical problems, or Dynamic Pro
 		url="https://atcoder.jp/contests/dp/tasks"
 		starred
 	>
-		Some tasks are beyond the scope of Gold.
+	</Resource>
+	 	Some tasks are beyond the scope of Gold.
```

Comment:
I think you switched the order of these two lines

Reply:
Now is fixed, in my host looks good.